Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -42,10 +42,10 @@ Some examples of EDSs include
 ## Implementation notes
 
 The normalised EDS `normEDS b c d n` is defined in terms of the auxiliary sequence
-`preNormEDS (b ^ 4) c d n`, which are equal when `n` is odd, and which differ by a factor of `b`
-when `n` is even. This coincides with the definition in the references since both agree for
-`normEDS b c d 2` and for `normEDS b c d 4`, and the correct factors of `b` are removed in
-`normEDS b c d (2 * (m + 2) + 1)` and in `normEDS b c d (2 * (m + 3))`.
+`preNormEDS (b ^ 4) c d n`, which are equal when `n` is odd, and which differ by a factor of an
+extra parameter `b` when `n` is even. This coincides with the definition in the references since
```

Comment:
I was suggesting this, since "extra parameter" is associated to a preNormEDS, and it's `b ^ 4` in this case.
```suggestion
`preNormEDS (b ^ 4) c d n`, which are equal when `n` is odd, and which differ by a factor of 
`b` when `n` is even. This coincides with the definition in the references since
```